public/documentation/getting_started/remote_debugging.md

```diff
@@ -1,6 +1,6 @@
 # Remote debugging
 
-Web applications running under Crosswalk can be debugged remotely using the [Chrome dev tools](https://developer.chrome.com/devtools/index). 
+Web applications running under Crosswalk can be debugged remotely using [Chrome dev tools](https://developer.chrome.com/devtools/index). 
 
```

Comment:
Yeah, I've had another look at that link, and they do seem to use an excessive number of "the" articles ... I guess instinctively I think of Chrome DevTools as a "kit" ... does work either way though.  Will change it back for consistency.